<launch>

    <node name="pose_graph" pkg="pose_graph" type="pose_graph" output="screen">
        <param name="use_ibow" type="bool" value="false" />
        <param name="use_mesh_maker" type="bool" value="false" />
        <param name="visualization_shift_x" type="int" value="0" />
        <param name="visualization_shift_y" type="int" value="0" />
        <param name="skip_cnt" type="int" value="0" />
        <param name="skip_dis" type="double" value="0" />
        <param name="pose_graph_save_path" type="string" value="/home/puyu/output/pose_graph/" />
        <param name="pose_graph_result_path" type="string" value="/home/puyu/output/" />
    </node>

    <node name="mobile_swarm" pkg="mobile_swarm" type="mobile_swarm" output="screen"/>

</launch>